docs/threshold_encryption.md

```diff
@@ -0,0 +1,41 @@
+# Threshold Encryption
+*Threshold encryption* is one of cryptosystem, if in order to decrypt an encrypted message or to sign a message, several 
+parties (more than some threshold number) must cooperate in the decryption or signature protocol.
+The message is encrypted using a `public key` and the corresponding `private key` is shared among the participating parties.
```

Comment:
> The message is encrypted using a `public key` and the corresponding `private key` is shared among the participating parties.

how about "~ and the set of `private key` is distributed to the participating parties" ?